avogadro/qtplugins/manipulator/manipulator.cpp

```diff
@@ -122,19 +124,39 @@ QUndoCommand * Manipulator::mouseReleaseEvent(QMouseEvent *e)
 QUndoCommand * Manipulator::mouseMoveEvent(QMouseEvent *e)
 {
   e->ignore();
+  if (!(m_pressedButtons & Qt::LeftButton))
```

Comment:
The manipulate code needs  a bunch of work - no zoom or rotate, but this is a start